test/E2E/AbstractE2ETestCase.php

```diff
@@ -95,7 +95,7 @@ protected function initializeComposer(string $path): string
     {
         $process = new Process(
             [
-                $this->executableFinder->find('composer'),
+                realpath($this->executableFinder->find('composer', 'composer')),
```

Comment:
Can you tell me what the issue is exactly? 
Why are you defaulting to the string 'composer' if the file cannot be found?
I try to avoid realpath as much as possible in this package, since it already caused a lot of issues.

Reply:
I have `...:vendor/bin:...` in the `$PATH` so the executable finder finds the first and tries to run that with a relative path.

Reply:
@janvernieuwe  : Does this fix the issue you had while running paratest as well? It might be related